evap/staff/templates/staff_course_form.html

```diff
@@ -59,24 +61,33 @@
             </tbody>
         </table>
 
+        {% if state == "inEvaluation" or state == "evaluated" or state == "reviewed" or state == "new" or state == "prepared" or state == "editorApproved" or course.can_staff_edit %}
```

Comment:
i'm sorry that i didn't completely think this through before. the states listed here are all states but `published` (`approved` is missing, but should actually be listed as well). so in all states in which the course can be edited we need this div. that means we can simply put a `{% if course.can_staff_edit %}` here.

The `Save` button can always be rendered (because we already checked for the `can_staff_edit`), the `Save and approve` button keeps the if with its three states.